js/main.js

```diff
@@ -1,36 +1,49 @@
 
-// runs this function once all DOM has been loaded
+
 window.onload = () => {
+    // selects elements function
+    const selectElement = (element) => document.querySelector(element);
+    const input = selectElement("#tntsearch-wrapper > input[type=text]");
+    const clearSearch = selectElement(".tntsearch-clear");
+    const overlay = selectElement("#overlay");
+    const toggleNav = selectElement("#sidebar-toggle");
+    let bool = false;
 
-    // variable declaration for query selectors
-    const input = document.querySelector("#tntsearch-wrapper > input[type=text]");
-    const clearSearch = document.querySelector(".tntsearch-clear");
-    const clickOverlay = document.querySelector("#overlay");
+    toggleNav.addEventListener("click", () => {
+
+        if (!bool){
+            selectElement("#sidebar").style.left = "251px";
+            selectElement("#body").style.left = "251px";
+            bool = true;
+        }
+        else if (bool) {
+            selectElement("#sidebar").style.left = "0px";
+            selectElement("#body").style.left = "0px";
+            bool = false;
+        }
+    });
 
     // listen event which checks if the user has typed anything into the search bar
     input.addEventListener("input", () => {
-        // Show result form
-        document.querySelector(".tntsearch-results").style.visibility = "visible";
+        selectElement(".tntsearch-results").style.visibility = "visible";
 
         // checks if there are 0 inputs in the search bar
         if (input.value === 0){
```

Comment:
I would expect `input.value` to always be a string however your comparing it to a number. I will always be false given your using the `===` operator. Are you asserting that the text box is empty? If so I'd suggest 

`input.value === ""` or `input.value.length === 0`

Reply:
Yeah im just checking if the text length is 0. It works but ill take your advice and change it to 'input.value.length'